packages/web-frontend/src/containers/OverlayDiv/components/NotFoundPage.js

```diff
@@ -0,0 +1,57 @@
+/*
+ * Tupaia
+ *  Copyright (c) 2017 - 2023 Beyond Essential Systems Pty Ltd
+ */
+import React from 'react';
+import styled from 'styled-components';
+import { Typography } from '@material-ui/core';
+import { PrimaryButton } from '../../../components/Buttons';
+import { goHome } from '../../../actions';
+import { connect } from 'react-redux';
+
+const Container = styled.div`
+  height: 500px;
+  width: 800px;
+  max-width: 100%;
+  max-height: 100%;
+  text-align: center;
+  padding-top: 20%;
+  padding-bottom: 30px;
+`;
```

Comment:
Potentially using `rems`/`ems`/percentages here instead of `px` would be better for future-proofing things